### PR TITLE
Create user

### PR DESCRIPTION
**Pull request by vujevits**
- design is rudimentary
- favicons are persisted for each post, I believe as intended

### CODE REVIEW

File: src/components/IdentitySettings.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+};
+
+const Header = (props: { onPressBack: () => void }): React.ReactElement<{}> => {
+    return (
+        <View style={styles.headerContainer}>
+            <View style={{ paddingLeft: 15 }}>
+                <TouchableOpacity
+                    onPress={props.onPressBack}
+                    activeOpacity={1.0}
+                    hitSlop={{
```
Comment (agazso) [inline src/components/IdentitySettings.tsx]: You can also use TouchableView here which has a default hitSlop of 30

File: src/reducers/index.ts [hunk 2]
```diff
@@ -35,12 +36,19 @@ const defaultSettings: Settings = {
     saveToCameraRoll: true,
 };
 
+const defaultAuthor: Author = {
+    name: '',
+    uri: '',
```
Comment (agazso) [inline src/reducers/index.ts]: Where is uri used and what is it supposed to mean?

File: src/reducers/index.ts [hunk 3]
```diff
@@ -35,12 +36,19 @@ const defaultSettings: Settings = {
...
+    faviconUri: '',
```
Comment (agazso) [inline src/reducers/index.ts]: Why is this called faviconUri? I think imageUri or avatarUri would be a better name.

File: src/components/IdentitySettings.tsx [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+                        top: 10,
+                        left: 10,
+                        bottom: 10,
+                        right: 10,
+                    }}
+                >
+                <Text style={{ color: '#007AFF' }}>
```
Comment (agazso) [inline src/components/IdentitySettings.tsx]: Can you make this a bit bigger? :)

File: src/components/IdentitySettings.tsx (hunk 1 above)
Comment (vujevits) [reply src/components/IdentitySettings.tsx]: fixed in 9be93f9328558afb5e61d224446f772a29cb0e32

File: src/components/IdentitySettings.tsx (hunk 4 above)
Comment (vujevits) [reply src/components/IdentitySettings.tsx]: fixed in 9be93f9328558afb5e61d224446f772a29cb0e32

**Pull request merged**